dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionRestRepositoryIT.java

```diff
@@ -519,6 +519,108 @@ public void createTest() throws Exception {
 
     }
 
+    @Test
+    public void createTestByAuthorizedUser() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and one collection.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .withLogo("ThisIsSomeDummyText")
+                                          .build();
+
+        ObjectMapper mapper = new ObjectMapper();
+        CollectionRest collectionRest = new CollectionRest();
+        // We send a name but the created collection should set this to the title
+        collectionRest.setName("Collection");
+
+        collectionRest.setMetadata(new MetadataRest()
+            .put("dc.description",
+                new MetadataValueRest("<p>Some cool HTML code here</p>"))
+            .put("dc.description.abstract",
+                new MetadataValueRest("Sample top-level community created via the REST API"))
+            .put("dc.description.tableofcontents",
+                new MetadataValueRest("<p>HTML News</p>"))
+            .put("dc.rights",
+                new MetadataValueRest("Custom Copyright Text"))
+            .put("dc.title",
+                new MetadataValueRest("Title Text")));
+
+        // ADD authorization on parent community
+        context.setCurrentUser(eperson);
+        authorizeService.addPolicy(context, parentCommunity, Constants.ADD, eperson);
```

Comment:
After this line we should have a `context.restoreAuthSystemState();` to turn the authorization system back "on". It only needs to be turned off temporarily to create the test data...after that, we need to be sure the tests themselves are obeying all authorization rules.